service/src/main/resources/api/service_openapi.yaml

```diff
@@ -1214,6 +1492,129 @@ components:
         - selectionData
         - uiConfig
 
+    ReviewV2:
+      type: object
+      properties:
```

Comment:
This needs to store the cohort it was created with. It will also need a creation timestamp though that won't be useful until cohorts have modified timestamps.

Reply:
To get a review, the caller needs to specify the study and cohort ids as URL parameters, so they should already have that information. That's why I didn't include it as part of the response object. That said, I don't have an objection to returning the request parameters as part of the response -- would that make it easier for you to handle in the UI?

RE timestamps: OK I'll add `cohort.lastModified` and `review.created` timestamp fields.

Reply:
On the backend, I'm planning to freeze the set of person ids included in the review when it was created, so that will use the cohort definition (i.e. the filter) as it exists at that time. I can also freeze the `selectionData` and `uiConfig`, though I don't expect the backend will need those again -- does the UI?

Reply:
Per offline conversation, UI does need these again so it can show a breakdown of the cohort at the time the review was created. Updated this to include a `CohortV2` reference.